docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+
+services:
+  remacs:
+    build: .
+    image: remacs-build:latest
+    working_dir: ${PWD}
+    tty: true
+    # Unfortunately dumping doesn't work without this
```

Comment:
Probably, but that will never get fixed, not until the old timers run things :/

Maybe one day remacs will fix it itself, or just remove the misfeature completely.